pages/learn/integration/pre-built-integrations/news-api-to-email-integration.js

```diff
@@ -222,12 +222,12 @@ export default function Learn({ samples, content }) {
               <Container>
                 <Row>
                   <Col xs={12} lg={6} style={{ fontSize: "18px" }}>
-                    <p>Integrating news headlines directly into Gmail through the NewsAPI and email integration carries
+                    <p>Integrating news headlines directly into Gmail through the News API and email integration carries
```

Comment:
NewsAPI is a name. Applies to all other places as well.

Reply:
It seems newsapi.org is using the name with spaces. so we also need to adhere to that?

<img width="1308" alt="image" src="https://github.com/ballerina-platform/ballerina-dev-website/assets/16300038/350ca64d-7555-49eb-a80a-25d7f1f3b16f">